abdulfatai360/banka
# #165646589 Implement feedback on project backend

#### What does this PR do?
Implement feedback on project backend

#### Description of Task to be completed?
- Make all inputs validation error messages show at once
- Make response error messages concise
- Disallow whitespace in password input fields

#### How should this be manually tested?
On Terminal:
- Run `git clone https://github.com/abdulfatai360/banka.git` to clone this repo onto your local machine
- Navigate to the repo directory by running `cd banka/`
- Ensure your working directory is clean by running  `git status`  command
- Get a copy of the PR by typing  `git fetch origin pull/54/head:ch-implement-backend-feedback-165646589`
- Now that you have a copy of the branch, switch to it using  `git checkout ch-implement-backend-feedback-165646589`
- Your directory will now be an exact copy of the PR
- Run `npm install` to install all project's dependencies onto your local machine
- Type `npm run build` to run the build script and populate the `dist/` directory containing transpiled code
- Run `npm start` command to start the development server

#### What are the relevant pivotal tracker stories?
[ #165646589](https://www.pivotaltracker.com/story/show/165646589)
